styles.css

```diff
@@ -95,3 +95,268 @@ div.box {
   margin-top: 20px;
   width: 100%;
 }
+
+h2 {
+  width: 220px;
+  height: 30px;
+  padding-left: 16px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 50px;
+  line-height: 100%;
+  color: #1a2236;
+}
+
+h3 {
+  width: 264px;
+  height: 16px;
+  left: 20px;
+  top: 960px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 20px;
+  line-height: 80%;
+  padding-left: 16px;
+  padding-top: 20px;
+  color: #fff;
+  margin-top: 0;
+  margin-bottom: 5px;
+}
+
+ul {
+  padding-left: 10px;
+}
+
+li {
+  display: inline-flex;
+  flex-direction: row;
+}
+
+.projects {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: auto auto auto auto auto auto auto auto auto auto auto auto auto;
```

Comment:
- [ ] [OPTIONAL] In this case, you might not really need to specify the number of rows. Grid has the ability to declare this implicitly so that it increases according to the number of your cards/elements. Great job, @Maedea9 :+1: